packages/katello/rubygem-redhat_access/rubygem-redhat_access.spec

```diff
@@ -0,0 +1,126 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name redhat_access
+%global plugin_name redhat_access
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
```

Comment:
The old package was called `rubygem-foreman-redhat_access`:
https://github.com/redhataccess/foreman-plugin/blob/98fc23806bb6832c69a2ede56c71525d8ba9f8f5/redhat-access/rubygem-foreman-redhat_access.spec#L17

Should we add `Obsoletes` and `Provides` for that name, so thjat upgrades are smoother?